### DISCUSSION TITLE
Cloud version works, self-hosted breaks. Confusion reigns.

### DISCUSSION THREAD

**Discussion by CBIweb** [tinymce/tinymce#7376] [General]
I'm a new TinyMCE user, and for now I'm working in my local dev environment while I get to know it. I'm on fully updated Windows 10 Home x64 with Apache, PHP7 and MariaDB.

At first I set up the cloud version, which works perfectly, but couldn't figure out how to add a plugin for hyperlinks.  So I downloaded the self-hosted version, and it stopped working. After looking through my own code and TMCE's code, I can't figure out why.

The _only_ difference between working and not working is the cloud API.  With the API, no problems at all.  But when I replace the API with `<script src="path/to/tinymce/js/tinymce.min.js"></script>` , it breaks (and yes, I do enter the proper path :) ).  The editor doesn't show up at all.

The API and the `path/to/` line are the only two things that change in the code, from what I can see.

Is there something within the self-hosted files that I need to tweak?  I assume the API pulls in everything from the cloud, that the self-hosted files contain.  But it seems I'm missing something somewhere.

Any help is appreciated.  Thanks in advance!

**Comment by GrtFS** [comment]
I have this exact issue.  I am applying TinyMCE to a Filemaker project.  I should be able to put tinymce.min.js (and other js/css files) into a global field and point to it, but it doesn't work.  It works when putting it on my domain-name sever, but otherwise, Tiny doesn't load.
